src/swf/parser/button.ts

```diff
@@ -31,22 +31,26 @@ module Shumway.SWF.Parser {
       if (character.eob)
         break;
       var characterItem = dictionary[character.symbolId];
-      release || characterItem || Debug.warning('undefined character in button ' + tag.id);
-      var cmd = {
-        symbolId: characterItem.id,
-        code: SwfTag.CODE_PLACE_OBJECT,
-        depth: character.depth,
-        flags: character.matrix ? PlaceObjectFlags.HasMatrix : 0,
-        matrix: character.matrix
-      };
-      if (character.stateUp)
-        states.up.push(cmd);
-      if (character.stateOver)
-        states.over.push(cmd);
-      if (character.stateDown)
-        states.down.push(cmd);
-      if (character.stateHitTest)
-        states.hitTest.push(cmd);
+      // The Flash Player ignores references to not defined symbols here. So should we.
```

Comment:
nit: s/not defined/undefined/

Also, same question as above: do we pick the symbols up later? In this case, I don't at all know what the Flash Player does, though.